### PR TITLE
Feature: Snippets

### PR DESCRIPTION
**Pull request by jam8608**

### CODE REVIEW

**Comment by jam8608** [comment]
That wasn't very epic.

File: README.md [hunk 1]
```diff
@@ -13,17 +15,18 @@ If you're interested in importing one of the custom actions found within our rep
...
+
+Have an awesome action you want to share? Here's how to get it on this repository for everyone to see. Before uploading your action, you might want to take a look at our **[Guidelines for Contributing](https://github.com/doddsy/atlas-custom-actions#guidelines-for-contributing)** to see if it meets our standards.
+* Fork this repository via the ![Fork](https://i.imgur.com/oazJQxh.png) button located in the top-right corner of this repository.
 * On the Atlas dashboard, select your action from the actions list on your [dashboard](https://atlasbot.xyz/).
-* Click the ![Export](https://i.imgur.com/dvXMSce.png) Export button to save the `.action` file to your local computer.
-* Rename your action file in correspondence to the **[Folder Format Requirements](https://github.com/atlasbot/community-actions#folder-format-requirements)** found at the bottom of this page.
+* Click the ![Export](https://i.imgur.com/dvXMSce.png) Export button to save the `.action` file to your device.
+* Rename your action file in correspondence to the **[Folder Format Requirements](https://github.com/doddsy/atlas-custom-actions#folder-format-requirements)** found at the bottom of this page.
```
Comment (doddsy) [inline README.md]: Why was the url changed to the old one? it should be `atlasbot/community-actions`, no?

File: Snippets/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# What Are Snippets?
+Snippets are small pieces of Atlas' TagScript code that, by themselves, do not make up a functional action. They are usually under 512 characters and can be copy/pasted into an existing action seamlessly by a semi-competent person who knows what they are doing. There is no special file format for uploading code snippets, see [Upload Requirements](https://github.com/atlasbot/community-actions/Snippets#upload-requirements) for more detailed information on how to properly contribute snippets.
```
Comment (doddsy) [inline Snippets/README.md]: *"can be copy/pasted into an existing action seamlessly by a semi-competent person who knows what they are doing"*
👍 nise

File: README.md (hunk 1 above)
Comment (jam8608) [reply README.md]: ¯\_(ツ)_/¯ beats me

**Pull request merged**